### PR TITLE
 📝  Add custom label to selection controls

### LINKED ISSUE #2231: Example of using custom label with Selection controls
Feedback from a project was that EDS only supports label on right side of the selection controls. Maybe make a short story for our selection controls where we use `Label` or `Typography` with text on left side and in sidesheet or something similar 🤔

Maybe better description that is just a styled `input` with `type=radio/checkbox` 

Suggested [example](https://codesandbox.io/s/serene-golick-wfr72c?file=/src/App.tsx)

### PR DESCRIPTION
**Pull request by martalalik**
Resolve #2231

### CODE REVIEW

File: packages/eds-core-react/src/components/Radio/Radio.stories.tsx [hunk 1]
```diff
@@ -116,6 +124,28 @@ export const AlternativeToLabel: Story<RadioProps> = () => (
 )
 AlternativeToLabel.storyName = 'Alternative to label'
 
+export const CustomLabel: Story<RadioProps> = () => {
+  return (
+    <div style={{ display: 'contents' }}>
```
Comment (oddvernes) [inline packages/eds-core-react/src/components/Radio/Radio.stories.tsx:129]: I don't think this is the correct usage of display: contents and the end result is a bit weird witht the distance between label and radio. How about display flex , align-items center.

**Review by oddvernes — CHANGES_REQUESTED** [review]

File: packages/eds-core-react/src/components/Radio/Radio.stories.tsx (hunk 1 above)
Comment (martalalik) [reply packages/eds-core-react/src/components/Radio/Radio.stories.tsx:129]: Pure provocation with `contents`, but yes you right.

**Comment by oddvernes** [comment]
I would perhaps use the example linked in the parent issue, since that was the use case someone was asking about

File: packages/eds-core-react/src/components/Radio/Radio.stories.tsx (hunk 1 above)
Comment (martalalik) [reply packages/eds-core-react/src/components/Radio/Radio.stories.tsx]: done

File: packages/eds-core-react/src/components/Radio/Radio.stories.tsx [hunk 2]
```diff
@@ -116,6 +124,28 @@ export const AlternativeToLabel: Story<RadioProps> = () => (
...
+      <Label label="Label" />
+      <Radio name="radio" />
```
[the remark this answers is not in the document]
Comment (martalalik) [reply packages/eds-core-react/src/components/Radio/Radio.stories.tsx]: done

**Review by oddvernes — APPROVED** [review]
LGTM

**Pull request merged**